commands/certmanager/apiversion.go

```diff
@@ -26,63 +35,31 @@ func CertGetApiVersionCmd() *core.Command {
 		InitClient: true,
 	})
 
-	cmd.Command.Flags().StringSlice(constants.ArgCols, nil, printer.ColsMessage(allCols))
+	cmd.Command.Flags().StringSlice(constants.ArgCols, nil, tabheaders.ColsMessage(defaultCertificateCols))
```

Comment:
`defaultCertificateCols` is printed as the set of all available columns, yet `allApiVersionCols` used for autocompleting the available columns